build.sbt

```diff
@@ -23,9 +23,9 @@ publishArtifact in Test := false
 pomIncludeRepository := { x => false }
 
 libraryDependencies ++= Seq(
-  "io.swagger" % "swagger-core" % "1.5.8",
-  "org.scalatest" %% "scalatest" % "2.2.5" % "test",
-  "com.fasterxml.jackson.module" %% "jackson-module-scala" % "2.4.2",
```

Comment:
Hi @pjfanning ;) ok, I see swagger-core would need to address https://github.com/swagger-api/swagger-core/issues/1937 in order for this to make progress, fair enough. I noticed jackson-module-scala has cross compiled for 2.12.0 since 2.8.4 (latest) so you'd either want to upgrade to that one in core or if you want to stay on the 2.7 branch we'd have to try and see whether jackson-scala-module 2.7.2+ can be cross compiled to 2.12.